src/cljs/rems/status_modal.cljs

```diff
@@ -1,67 +1,118 @@
-      [:div
-       (when @state [component @state])
-       [:button.btn.btn-secondary {:on-click #(reset! state (assoc opened-state :on-close on-close))} "Open modal"]])))
+(defn set-pending! [& [opts]]
+  (rf/dispatch [::set-state (deep-merge {:open? true}
+                                        opts)]))
+(defn set-success! [opts]
+  (rf/dispatch [::merge-state (deep-merge {:open? true
+                                           :result {:success? true}}
+                                          opts)]))
+(defn set-error! [opts]
+  (rf/dispatch [::merge-state (deep-merge {:open? true
+                                           :result {:success? false}}
+                                          opts)]))
 
 (defn guide
   []
   [:div
    (component-info status-modal)
-   (example "status-modal that opens as pending"
-            [example-wrapper {:opened-state {:status :pending
-                                             :description "Pending modal"}
-                              :component status-modal}])
-   (example "status-modal that opens as saved"
-            [example-wrapper {:opened-state {:status :saved
-                                             :description "Saved modal"}
-                              :component status-modal}])
-   (example "status-modal that opens as failed"
-            [example-wrapper {:opened-state {:status :failed
-                                             :error {:status 404
-                                                     :status-text "Not found"}
-                                             :description "Failed modal"}
-                              :component status-modal}])])
+   (example "status-modal while result is pending"
+            [status-modal {:open? true
+                           :shade? false
+                           :title "Pending"
+                           :content [:p "We are experiencing unexpected slowness"]}])
+   (example "status-modal for success"
+            [status-modal {:open? true
+                           :result {:success? true}
+                           :shade? false
+                           :title "Success"
+                           :content [:p "This was a great success for all!"]}])
+   (example "status-modal for result with a single error"
+            [status-modal {:open? true
+                           :result {:error {:status 404
+                                            :status-text "Not found"}}
+                           :shade? false
+                           :title "Error"
+                           :content [:p "This did not go as planned"]}])
+   (example "status-modal for result with errors"
+            [status-modal {:open? true
+                           :result {:errors [{:type :t.form.validation/errors}]}
+                           :shade? false
+                           :title "Errors"
+                           :content [:p "You should check the errors"]}])])
+
+(defn common-pending-handler! [title]
```

Comment:
I missed these at first because they were _after_ the guide

also, (short) docs please

Reply:
Yes indeed!